include/mbedtls/ssl.h

```diff
@@ -3775,13 +3777,15 @@ void MBEDTLS_DEPRECATED mbedtls_ssl_conf_sig_hashes(mbedtls_ssl_config *conf,
 #endif /* !MBEDTLS_DEPRECATED_REMOVED && MBEDTLS_SSL_PROTO_TLS1_2 */
 
 /**
- * \brief          Configure allowed signature algorithms for use in TLS 1.3
+ * \brief          Configure allowed signature algorithms
  *
  * \param conf     The SSL configuration to use.
  * \param sig_algs List of allowed IANA values for TLS 1.3 signature algorithms,
  *                 terminated by \c MBEDTLS_TLS1_3_SIG_NONE. The list must remain
```

Comment:
Could this be "terminated by `MBEDTLS_TLS_SIG_NONE`"?

Reply:
This should be a link:
```suggestion
 *                 terminated by #MBEDTLS_TLS1_3_SIG_NONE. The list must remain
```

There is no `MBEDTLS_TLS_SIG_NONE` in `ssl.h`. It exists in an internal header.

Reply:
Should we remove MBEDTLS_TLS1_3_SIG_NONE?

Reply:
> Could this be "terminated by `MBEDTLS_TLS_SIG_NONE`"?

No.  the field is different in TLS1.2 and TLS1.3. This name means it follows TLS1.3 spec.

In TLS1.2,  `SignatureAlgorithms` is a filed of `SignatureAndHashAlgorithm`. But TLS1.3 renames it to SignatureSchemeList. And the first 3 characters are same.
```
/* RFC 5446 7.4.1.4.1 */
   enum {
          ...
      } HashAlgorithm;

      enum { ...}
        SignatureAlgorithm;

      struct {
            HashAlgorithm hash;
            SignatureAlgorithm signature;
      } SignatureAndHashAlgorithm;

      SignatureAndHashAlgorithm
        supported_signature_algorithms<2..2^16-2>;
/* RFC 8446 4.2.3 */
    enum {
          ...
          /* Reserved Code Points */
          private_use(0xFE00..0xFFFF),
          (0xFFFF)
      } SignatureScheme;

      struct {
          SignatureScheme supported_signature_algorithms<2..2^16-2>;
      } SignatureSchemeList;
```